test/python/opflow/test_gradients.py

```diff
@@ -910,6 +913,110 @@ def test_vqe(self):
         result = vqe.compute_minimum_eigenvalue(operator=h2_hamiltonian)
         np.testing.assert_almost_equal(result.optimal_value, h2_energy, decimal=0)
 
+    @slow_test
```

Comment:
I don't think these tests are run at all if they have the `@slow_test` decorator. If we add them, then we should make them fast so they are feasible to run 😜

Reply:
Ok :)
But I didn't add the other `@slow_test` decorators. Do you want me to remove them or shall I leave them?

Reply:
Ideally, we would make the test so fast that we don't need avoid running it because it's slow. Maybe we can think of a sufficient test that runs in like 1-5s? We can also brainstorm on Slack 🙂